docs/release-notes/index.md

```diff
@@ -8,6 +8,58 @@ mapped_pages:
+
+#### User Password Generation (Enterprise feature)
+
+ECK will now generate longer passwords by default for the administrative user of each Elasticsearch cluster. The password is 24 characters in length by default (can be configured to a maximum of 72 characters), incorporating alphabetic and numeric characters, to make password complexity stronger.
+
+### Features and enhancements  [elastic-cloud-kubernetes-320-features-and-enhancements]
+
+- Enable certificate reloading for stack monitoring Beats [#8833](https://github.com/elastic/cloud-on-k8s/pull/8833) (issue: [#5448](https://github.com/elastic/cloud-on-k8s/issues/5448))
+- Allow configuration of file-based password character set and length [#8817](https://github.com/elastic/cloud-on-k8s/pull/8817) (issues: [#2795](https://github.com/elastic/cloud-on-k8s/issues/2795), [#8693](https://github.com/elastic/cloud-on-k8s/issues/8693))
+- Automatically set GOMEMLIMIT based on CGroup memory limits [#8814](https://github.com/elastic/cloud-on-k8s/pull/8814) (issue: [#8790](https://github.com/elastic/cloud-on-k8s/issues/8790))
+- Introduce granular PodDisruptionBudgets based on node roles [#8780](https://github.com/elastic/cloud-on-k8s/pull/8780) (issue: [#2936](https://github.com/elastic/cloud-on-k8s/issues/2936))
+
+### Fixes  [elastic-cloud-kubernetes-320-fixes]
+
+- Gate advanced Fleet config logic to Agent v8.13 and later [#8869](https://github.com/elastic/cloud-on-k8s/pull/8869)
+- Set CONFIG_PATH and STATE_PATH to the same directory in Fleet mode [#8856](https://github.com/elastic/cloud-on-k8s/pull/8856) (issue: [#8819](https://github.com/elastic/cloud-on-k8s/issues/8819))
+- Do not set credentials label on Kibana config secret [#8852](https://github.com/elastic/cloud-on-k8s/pull/8852) (issue: [#8839](https://github.com/elastic/cloud-on-k8s/issues/8839))
+- Allow elasticsearchRef.secretName in Kibana helm validation [#8822](https://github.com/elastic/cloud-on-k8s/pull/8822) (issue: [#8816](https://github.com/elastic/cloud-on-k8s/issues/8816))
+
+### Documentation improvements  [elastic-cloud-kubernetes-320-documentation-improvements]
+
+- Update Logstash recipes from to filestream input [#8801](https://github.com/elastic/cloud-on-k8s/pull/8801)
+- Recipe for exposing Fleet server to outside of the Kubernetes cluster [#8788](https://github.com/elastic/cloud-on-k8s/pull/8788)
+- Clarify secretName restrictions [#8782](https://github.com/elastic/cloud-on-k8s/pull/8782)
+- Update ES_JAVA_OPTS comments and explain auto-heap behavior [#8753](https://github.com/elastic/cloud-on-k8s/pull/8753)
+
+### Miscellaneous  [elastic-cloud-kubernetes-320-miscellaneous]
+
+- chore(deps): update go [#8837](https://github.com/elastic/cloud-on-k8s/pull/8837)
+- fix(deps): update all ungrouped dependencies [#8836](https://github.com/elastic/cloud-on-k8s/pull/8836)
+- fix(deps): update all ungrouped dependencies [#8826](https://github.com/elastic/cloud-on-k8s/pull/8826)
```

Comment:
This feels less useful now that the updates are grouped. (see the same section for 3.1 as a comparison: https://www.elastic.co/docs/release-notes/cloud-on-k8s#elastic-cloud-kubernetes-310-miscellaneous)

<img width="879" height="553" alt="image" src="https://github.com/user-attachments/assets/077b73c7-3fb6-44b9-b5af-ee094f933ba4" />